pyiceberg/catalog/dynamodb.py

```diff
@@ -81,6 +82,10 @@
 ITEM = "Item"
 
 
+@deprecated(
+    deprecated_in="0.6.0",
```

Comment:
`0.6.0` was already released, you can replace this with `0.6.1` or the next minor version `0.7.0`

Reply:
Hey @hussein-awala, yeah we came across this in the java pr as well: https://github.com/apache/iceberg/pull/9783#issuecomment-1961679888. but since we don't know if there will be an RC we found it easier to deprecate against the current version